packages/lab/src/__tests__/__e2e__/list-next/ListNext.cy.tsx

```diff
@@ -0,0 +1,317 @@
+                {ITEMS[1].label}
+              </ListItemNext>
+              <ListItemNext value={ITEMS[2].value}>
+                {ITEMS[2].label}
+              </ListItemNext>
+            </>
+          }
+        />
+      );
+
+      cy.findByRole("option", { name: ITEMS[1].label }).should(
+        "have.attr",
+        "aria-disabled",
+        "true"
+      );
+    });
+
+    it("should not be focusable on mouse hover", () => {
+      cy.mount(
+        <SingleSelectList
+          children={
+            <>
+              <ListItemNext value={ITEMS[0].value}>
+                {ITEMS[0].label}
+              </ListItemNext>
+              <ListItemNext value={ITEMS[1].value} disabled>
+                {ITEMS[1].label}
+              </ListItemNext>
+              <ListItemNext value={ITEMS[2].value}>
+                {ITEMS[2].label}
+              </ListItemNext>
+            </>
+          }
+        />
+      );
+
+      cy.findByRole("option", { name: ITEMS[1].label })
+        .realHover()
+        .should("not.have.class", "saltListItemNext-focused");
+    });
+  });
+
+  // Keyboard navigations for uncontrolled single select list
+  describe("Keyboard navigation and interactions", () => {
+    it("should select list item on Space or Enter key", () => {
+      cy.mount(<SingleSelectList />);
+
+      cy.findByRole("listbox").focus();
+      cy.realPress("ArrowDown");
+      cy.realPress("Space"); // select second list item
+      cy.findByRole("option", { name: ITEMS[1].label }).should(
+        "have.class",
+        "saltListItemNext-focused"
+      );
+      cy.findByRole("option", { name: ITEMS[0].label }).should(
+        "not.have.attr",
+        "aria-selected"
+      );
+    });
+
+    it("should focus and select first list item on first list keyboard focus", () => {
+      cy.mount(<SingleSelectList />);
+
+      cy.findByRole("listbox").focus();
+      cy.findByRole("option", { name: ITEMS[0].label }).should(
+        "have.attr",
+        "aria-selected",
+        "true"
+      );
+      cy.findByRole("option", { name: ITEMS[0].label }).should(
+        "have.class",
+        "saltListItemNext-focused"
+      );
+    });
+
+    it("should re-focus on previously focused item when an item was previously focused", () => {
+      cy.mount(<SingleSelectList />);
+
+      cy.findByRole("listbox").focus();
+      cy.realPress("ArrowDown");
+      cy.findByRole("option", { name: ITEMS[1].label }).should(
+        "have.class",
+        "saltListItemNext-focused"
+      );
+      cy.findByRole("listbox").blur(); // remove focus
+      cy.findByRole("listbox").focus(); // focus again
+      // focus should be back on the previously focused item
+      cy.findByRole("option", { name: ITEMS[1].label }).should(
+        "have.class",
+        "saltListItemNext-focused"
+      );
+    });
+
+    it("should re-focus on previously selected item when an item was previously selected, unless focus has moved", () => {
```

Comment:
i don't think we need this test. we only check for the focus index, so selected has no impact